packages/components/src/components/tag/tag.scss

```
@@ -100,51 +102,98 @@
 		}
 	}
 
-	label,
-	span {
+	a {
+		@extend %default-transition;
+		@include color-variants.bg-transparent-interactive(false);
+		display: inline-flex;
+		text-decoration: none;
+
+		&::after {
+			--db-icon-margin-start: 0;
+			content: "" !important;
+		}
+	}
+
+	.db-link:hover,
+	.db-link:active {
+		color: inherit;
+	}
+
+	label {
 		@extend %db-overwrite-font-size-sm;
 		margin-inline-start: 0;
+		z-index: 1;
+	}
+
+	&:not(:has(dbbutton)):not(:has(db-button)):has(
+			button:not(.db-tab-remove-button)
+		),
+	&:has(dbbutton):has(dbbutton:not(.db-tab-remove-button)),
+	&:has(db-button):has(db-button:not(.db-tab-remove-button)),
```

```suggestion
	&:has(dbbutton:not(.db-tab-remove-button)),
	&:has(db-button:not(.db-tab-remove-button)),
```
aren't those redundant ?